src/_product-updates/2024-05-21-single-change.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Single Change
+tags: [New]
+image: /images/changelog/single-change.png
+---
+
+![single-change.png](/images/changelog/single-change.png)
+
+The changelog allows the consumers of your API documentation to track changes as your documentation evolves. We have recently added several new features to the changelog, such as the [changelog diff](https://docs.bump.sh/product-updates/2023/11/02/changelog-diff/) to track for example the differences between two changes.
+
+Today, we are adding a new feature to the changelog: individual pages for each change in your documentation. Each release of your documentation generates a new entry in the changelog that now has its own page and unique URL. This allows you to easily share a link directly pointing to specific changes with members of your organization or your ecosystem.
```

Comment:
I think we can add a small reminder here that they can add markdown content to this page to present/explain the API change, or write a full announcement.